src/Domain/WsServer/Plugins/Latest/LatestWsServerPlugin.php

```diff
@@ -67,104 +169,60 @@ private function latest(): PromiseInterface
             $clientInfoPerSessionContainer = $clientInfoPerSessionContainer->only($gameSessionId);
         }
         $promises = [];
-        foreach ($clientInfoPerSessionContainer as $clientInfoContainer) {
-            foreach ($clientInfoContainer as $connResourceId => $clientInfo) {
-                $accessTimeRemaining = 0; // not used
-                if (false === $this->getClientConnectionResourceManager()->getSecurity($connResourceId)->validateAccess(
-                    Security::ACCESS_LEVEL_FLAG_FULL,
-                    $accessTimeRemaining,
-                    $this->getClientConnectionResourceManager()->getClientHeaders($connResourceId)[
-                        ClientHeaderKeys::HEADER_KEY_MSP_API_TOKEN
-                    ]
-                )) {
-                    // Client's token has been expired, let the client re-connected with a new token
-                    $this->addOutput('Client\'s token has been expired, let the client re-connected with a new token');
-                    $this->getClientConnectionResourceManager()->getClientConnection($connResourceId)->close();
-                    continue;
+        foreach ($clientInfoPerSessionContainer as $gameSessionId => $clientInfoContainer) {
+            $game = new Game();
+            $game->setGameSessionId($gameSessionId);
+            $gameConfigValues = $game->GetGameConfigValues();
```

Comment:
in the Tick plugin >> GameTick.php you copied over a function from the server API a while ago called AreSimulationsUpToDate(). Do you want to reuse that here somehow? If not you could also consider reusing Simulations >> GetConfiguredSimulationTypes()